charts/incubator/openbooks/values.yaml

```diff
@@ -0,0 +1,43 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: evanbuss/openbooks
+  # -- image tag
+  tag: sha-ef086d3
```

Comment:
Too bad the releases from `https://github.com/evan-buss/openbooks/releases` aren't reflected in the Docker tags 😞 Maybe in an upcoming version we can build/host a properly tagged image under the kah repo